### DISCUSSION TITLE
PIP-1: PIP Purpose and Guidelines

### DISCUSSION THREAD

**Discussion by ja7ad** [pactus-project/PIPs#135] [PIPs]
PIP-1: [PIP Purpose and Guidelines](https://pips.pactus.org/PIPs/pip-1)

**Comment by b00f** [comment]
I suggest to add "Wallet" category for wallet PIPs, including 
- https://pips.pactus.org/PIPs/pip-11 
- https://pips.pactus.org/PIPs/pip-13

**Reply by ja7ad** [reply]
https://github.com/pactus-project/PIPs/pull/147

**Reply by b00f** [reply]
Don;t forget to update pip-1 (add wallet there under categories)

**Comment by b00f** [comment]
We need to write about attachments in PIPs
